daemon/src/model.rs

```diff
@@ -302,4 +609,60 @@ mod tests {
 
         assert!(past_event.has_likely_occured());
     }
+
+    #[test]
+    fn algebra_with_usd() {
+        let usd_0 = Usd::new(dec!(1.234));
+        let usd_1 = Usd::new(dec!(9.876));
+
+        let usd_sum = usd_0 + usd_1;
+        let usd_diff = usd_0 - usd_1;
+        let half = usd_0 / 2;
+        let double = usd_1 * 2;
+
+        assert_eq!(usd_sum.0, dec!(11.110));
+        assert_eq!(usd_diff.0, dec!(-8.642));
+        assert_eq!(half.0, dec!(0.617));
+        assert_eq!(double.0, dec!(19.752));
+    }
+
+    #[test]
+    fn usd_for_1_btc_buys_1_btc() {
+        let usd = Usd::new(dec!(61234.5678));
+        let price = Price::new(dec!(61234.5678)).unwrap();
+        let inv_price = InversePrice::new(price).unwrap();
+        let res_0 = usd / price;
+        let res_1 = usd * inv_price;
+
+        assert_eq!(res_0, Amount::ONE_BTC);
+        assert_eq!(res_1, Amount::ONE_BTC);
+    }
+
+    #[test]
+    fn leverage_does_not_alter_type() {
+        let usd = Usd::new(dec!(61234.5678));
+        let leverage = Leverage::new(3).unwrap();
+        let res = usd * leverage / leverage;
+
+        assert_eq!(res.0, usd.0);
+    }
+
+    #[test]
+    fn test_liquidation_as_expected() {
```

Comment:
```suggestion
    fn test_algebra() {
```

Reasoning: This test is not about testing the liquidation price but just algebra and constructing types. The liquidation price is just used as an example, but no code calculating the liquidation price is tested.

Reply:
That is actually (more or less) what I had and was told to change it to (more or less) the name that is there.  I'll change it again as I prefer your suggestion, but perhaps we should create a style-guide for this sort of thing as there is clearly some inconsistency happening?